assignments/objects.js

```diff
@@ -49,16 +97,54 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+class Parent {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+    this._child = null; 
+  }
+ 
+  Speak(phrase) {
+    return this.name + " said " + phrase;
+  }
+
+  get Child() {
+    return this._child;
+  }
+  
+  set Child(value) {
+    this._child = value;
+  }
+  
+}
+
+class Child extends Parent {
+  constructor(name, age) {
+    super(name,age);
+  }
+
+  Speak(phrase) {
+    return this.name + " said " + phrase;
+  }
+}
+
 
 // Log the parent object's name
 
-// Log the child's age
+const susan = new Parent("Susan", 70);
 
+// Log the child's age
+const george = new Child("George", 50);
+susan.Child = george; 
 // Log the name and age of the grandchild
+const sam = new Child("Sam", 30);
+george.Child = sam; 
 
 // Have the parent speak
+console.log(susan.Speak("this"));
 
 // Have the child speak
-
+console.log(george.Speak("that"));
```

Comment:
There is an old technique with binding `this` that is not used and frowned upon that involves defining a variable called  `that` - I think you are just trying to use string examples here but just to be safe try other strings when using examples in these kind of things just to avoid confusion.